server/src/main/java/com/ncc/neon/services/ClusterService.java

```diff
@@ -208,6 +224,62 @@ public TabularQueryResult cluster(TabularQueryResult tabularQueryResult) {
         return newData;
     }
 
+    /**
+     * Calculates the spacing between the different bins in the new data set results for the LatLon ClusterType.
+     *
+     * @param fieldNames the keys that correspond with the field names
+     * @param longCount how many longitude bins there should be in the new data set
+     * @param clusters clusters specified by the cluster clause if available
+     * @return the new data set ready to be filled
+     */
+    private List<Map<String, Object>> getNewLatLonDataBins(List<String> fieldNames, BigDecimal longCount, List<List<Object>> clusters) {
+        List<Map<String, Object>> newData = new ArrayList<>();
+
+        String latKey = fieldNames.get(0);
+        String longKey = fieldNames.get(1);
+
+        BigDecimal latCount = longCount.divide(new BigDecimal(2));
+        //TODO: round this?
+        BigDecimal totalCount = longCount.multiply(latCount);
```

Comment:
Both counts should be integers, right?  So I don't think there's a need to round them.

Reply:
I actually forgot this was in here!

Reply:
Fixed.